app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
I know you just imported this but I think it would be worth trying to convert to a Constraint layout instead of a bunch of nested layouts.

Reply:
I might go about that on a layout specific PR